STYLE_GUIDE.md

```diff
@@ -37,4 +37,9 @@ BAR = [
 # pylint: disable=protected-access
 foo._protected_member
 # pylint: enable=protected-access
-```
\ No newline at end of file
+```
+
+
+##### Python Special Cases Conventions
+
+Follow the guidance in the [TensorFlow Style Guide - Conventions sections](https://www.tensorflow.org/community/contribute/code_style#tensorflow_conventions_and_special_uses).
```

Comment:
nit: maybe drop the word "sections" from the link?

Reply:
done